hack/remove-calico-policy/override-policy.md

```diff
@@ -19,17 +19,33 @@ ingress and egress traffic, overriding any other Network Policy that has been co
 
 #### Override Calico policy to allow all traffic
 
-Create a `GlobalNetworkPolicy` resource by running the following command:
+Create a `GlobalNetworkPolicy` (`SystemNetworkPolicy` for Calico `v2.4.x` or older) resource by running the following command:
```

Comment:
Maybe just reword this to something like:  "Create the policy-override resource by running..."